pyacq/core/stream.py

```diff
@@ -284,10 +309,14 @@ def compress_blosclz4(self, index, data):
         data = blosc.pack_array(data, cname='lz4')
         return index, data
     
-    def send(self, index, data):
-        # TODO deal when data is nparray and self.copy is False and a.flags['C_CONTIGUOUS']=False
-        for f in self.funcs:
-            index, data = f(index, data)
+    def send(self, index, data, autoswapaxes = True):
+        if isinstance(data, np.ndarray):
+            if autoswapaxes and self.params['timeaxis']!=0:
+                data = data.swapaxes(0, self.params['timeaxis'])
+            if not data.flags['C_CONTIGUOUS']:
+                data = data.copy()
```

Comment:
What append if 'F_CONTINOUS' the receivers get a transposed array no ?
We have on the streamspec one degree of freedom  (for the axis) If we had a second possibility to have C or F this make things more complicated.
The receiver perform a np.frombuffer so it needs to known everything (axis and F/C) and this has to fixed in advance. I reject F_CONINUOUS to simplify. WHat do you think ?